tools/build_script_generator/cmake/resources/Doxygen.cmake.in

```diff
@@ -0,0 +1,11 @@
+function(enable_doxygen)
+  option(ENABLE_DOXYGEN "Enable doxygen doc builds of source" OFF)
+  if(ENABLE_DOXYGEN)
+    set(DOXYGEN_CALLER_GRAPH YES)
+    set(DOXYGEN_CALL_GRAPH YES)
+    set(DOXYGEN_EXTRACT_ALL YES)
+    find_package(Doxygen REQUIRED dot)
+    doxygen_add_docs(doxygen-docs ${PROJECT_SOURCE_DIR})
+
+  endif()
+endfunction()
```

Comment:
We have "deprecated" the use of doxygen some time ago and moved forward to [Doxypress](https://www.copperspice.com/documentation-doxypress.html) (with proper C++14/17/20 support), see #281.

Building documentation is already handled build-tool independent by lbuild module [`modm:docs`](https://modm.io/reference/module/modm-docs/).

Reply:
Ok. is't only an inheritance from another project, it can be changed.